pkg/registry/apis/secret/contracts/inline.go

```diff
@@ -6,6 +6,9 @@ import (
 	common "github.com/grafana/grafana/pkg/apimachinery/apis/common/v0alpha1"
 )
 
+// InlineSecureValueSupport allows resources to manage secrets inline
+//
+//go:generate mockery --name InlineSecureValueSupport --structname MockInlineSecureValueSupport --inpackage --filename inline_mock.go --with-expecter
```

Comment:
See https://github.com/grafana/grafana/blob/main/pkg/registry/apis/secret/inline_mock.go there's already the mock there and the mockery comment, so you can remove this comment and the file below 😃